srml/democracy/src/lib.rs

```diff
@@ -431,29 +615,44 @@ impl<T: Trait> Module<T> {
 		Self::deposit_event(RawEvent::Executed(index, ok));
 	}
 
+	/// Finds the proposal that will be the next referendum, unreserves the currency of
```

Comment:
```suggestion
	/// Find the proposal that will be the next referendum, unreserves the currency of
```

Reply:
nitpick: just for consistency, since we're saying `Tally` not `Tallying` in other doc comments